workloads/templates/vm-creation.md

```diff
@@ -6,7 +6,7 @@ The [OpenShift's template mechanism](https://docs.openshift.org/latest/dev_guide
 
 In order to create a virtual machine via OpenShift CLI, you need to provide a template defining the corresponding object and its metadata.
```

Comment:
This whole thing needs an update. We already ship the templates the user can use. Francesco's patch describes the format https://github.com/kubevirt/user-guide/pull/151 and the templates are released here https://github.com/kubevirt/common-templates/releases

So the user does not have to create their own template as described below, they can take one of the templates we released for them. I know you mention it at the bottom, but we should place it more prominently so people actually notice. Otherwise nobody will use them.

Reply:
Maybe at least good idea to mentioned regarding cli opportunity? I was inspired by OpenShift/Origin guides.

Reply:
Martin raised a good point. I think you can maybe just add references to the openshift docs, no need to replicate it there.